google/generativeai/types/content_types.py

```diff
@@ -723,20 +746,36 @@ def _make_tool(tool: ToolType) -> Tool:
             code_execution = tool.code_execution
         else:
             code_execution = None
```

Comment:
Do all the tests still pass without the `if "code_execution" in tool:/else` above?